app/services/document_update_params.rb

```diff
@@ -12,11 +12,11 @@ def initialize(document)
   end
 
   def update_params(params)
-    path_title = params[:document][:title].chomp[0...TITLE_SLUG_MAX_LENGTH]
+    path_title = params[:document][:title].squish[0...TITLE_SLUG_MAX_LENGTH]
```

Comment:
I've not seen `squish` before but I like it 😄

Reply:
Oh yes that's a new one for me too